### ISSUE TITLE
Code causes multiple HTML validation errors

### ISSUE THREAD

**Issue by docevans** [canha42/pinterest-pin-it#27]
http://www.firsttech.com/?page_id=27

It doesn't like the ampersand between Url and image address (right before the word media):
 span class="xc_pin" onclick="pin_this(event, 'http://pinterest.com/pin/create/button/?url=http://www.firsttech.com/?page_id=2723&media=wp-content/uploads/2010/10/facetofaceSEMINARS.png&description=Free Seminar Schedule')"
            </span>

It also errors on the ampersand before the word description.

**Comment by chvillanuevap** [comment]
In ppibfi_pinterest.php, you need to change:

``` php
            <span class="xc_pin" onclick="pin_this(event, \''.
            $pinterest_base_url.
            '?url='.esc_url($post_url).
            '&media=%2$s.%3$s'.'&description='.$post_title.'\')">
            </span>
```

To:

``` php
            <span class="xc_pin" onclick="pin_this(event, \''.
            $pinterest_base_url.
            '?url='.esc_url($post_url).
            '&amp;media=%2$s.%3$s'.'&amp;description='.$post_title.'\')">
            </span>
```

**Comment by canha42** [comment]
Thanks @chvillanuevap - will be added to 1.1 and tested!

**Comment by canha42** [comment]
Tested and running. Available in version 1.1
